tests/integration/functional/test_zen_server_api.py

```diff
@@ -0,0 +1,75 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from typing import Tuple
+
+import pytest
+import requests
+
+from zenml.client import Client
+from zenml.constants import API, STACKS, USERS, VERSION_1
+
+SERVER_START_STOP_TIMEOUT = 30
+
+
+@pytest.fixture
+def rest_api_auth_token() -> Tuple[str, str]:
+    """Get an authentication token from the server.
+
+    Yield:
+        The server's REST API base URL and an authentication token.
+    """
+    from zenml.zen_stores.rest_zen_store import RestZenStore
+
+    zen_store = Client().zen_store
+    assert isinstance(zen_store, RestZenStore)
+
+    yield zen_store.url, zen_store._get_auth_token()
+
+
+def test_get_stack_endpoints(rest_api_auth_token):
```

Comment:
This is cool, but IMO not necessary once we can run the ZenStore tests with a REST ZenStore

Reply:
True. I didn't create these tests through, just refurbished them from the unit test area.